ModernWallet/Classes/ViewController/DimViewController/AssetDisclaimerViewController.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  AssetDisclaimerViewController.swift
+//  ModernMoney
+//
+//  Created by Georgi Stanev on 10.05.18.
+//  Copyright © 2018 Lykkex. All rights reserved.
+//
+
+import UIKit
+import RxCocoa
+import RxSwift
+import WalletCore
+import WebKit
+
+class AssetDisclaimerViewController: UIViewController, UIWebViewDelegate {
+
+    
+    @IBOutlet weak var acceptedButton: UIButton!
+    @IBOutlet weak var `continue`: UIButton!
+    @IBOutlet weak var cancel: UIButton!
+    @IBOutlet weak var disclaimerWebView: UIWebView!
```

Comment:
Note: It is highly recommended to use **WKWebView** instead of **UIWebView** (source: [HERE](https://developer.apple.com/documentation/webkit/wkwebview) )